modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.
+* The virtual machine you want to access is running.
+
+.Procedure
+
+. Modify the `~/.ssh/config` on your client machine to include the following text:
++
+[source,terminal]
+----
+Host *.cluster

```

Comment:
This looks good to me. i was also checking https://kubevirt.io/user-guide/virtual_machines/accessing_virtual_machines/#using-virtctl-as-proxy. 
Goto "To provide easier access to arbitrary virtual machines", maybe we might want to be in sync with this.(Just a thought)

Reply:
Actually disregard my comment above (I was hyper-focused on the wording in the link as opposed to the example config). I agree this should have some sort of clarification to it. I'll look into it and request a review again shortly.

Reply:
Hi @fabiand do you mind taking a look at the recommendation from @geetikakay above? Specifically I'd like to ask about two things:

1. In our example, we use `Host *.cluster` (line 22) while the u/s docs say `Host vmi/*`. Do you have an opinion on which should be used?
2. Also in our example, we use `.cluster` (line 30) while the u/s docs say `.mynamespace`. Any opinions on this as well?

Reply:
I think the cluster settings are correct. U/s docs would not assume that the user is a cluster admin.
@phoracek  What do you think? 

Also, please see my comment below, re: indenting the 2nd line in the ssh config. thanks.